src/utils/formValidations.js

```diff
@@ -32,3 +32,32 @@ export const validatePassword = (password) => {
   return [true, ""];
```

Comment:
it would be good that this formValidations.js is inside a folder referred to the task you are doing, since there can be more validations to do and it can be confused.

Reply:
It is located in the utils folder because they are validations that can be applied to various forms, such as login and signup.